README.md

```diff
@@ -4,6 +4,15 @@ A tool to enumerate, list and monitor USB devices connected to a Linux host.
 
 The tool will print out a list of connected devices with USB path, vendor, serial and connection speed. Following that it will start monitoring for udev events such as connect or disconnect and print them with a timestamp. Uses `libudev`.
 
+Requirement
```

Comment:
This section is too long. Please make it much shorter. For example `requires libudev-dev`. That should be enough.

Reply:
Yes!
You're right, i'll fix it.